Source/Core/DolphinQt2/Resources.cpp

```diff
@@ -13,25 +17,37 @@ QList<QPixmap> Resources::m_countries;
 QList<QPixmap> Resources::m_ratings;
 QList<QPixmap> Resources::m_misc;
 
+QString Resources::GetPreferredIconScale(int max)
```

Comment:
Ah, okay. I still don't understand this `max` argument. Why not always try to load the best pixel ratio image, and fallback if it doesn't exist, just like `QIcon` does internally?